tests/integration/generated/test_debug2_chrysalis.cfg

```diff
@@ -0,0 +1,51 @@
+[default]
+case = v2.LR.historical_0201
+constraint = ""
+dry_run = "False"
+environment_commands = ""
+input = "/lcrc/group/e3sm/ac.forsyth2//E3SMv2/v2.LR.historical_0201"
+input_subdir = archive/atm/hist
+mapping_file = "map_ne30pg2_to_cmip6_180x360_aave.20200201.nc"
+# To run this test, edit `output` and `www` in this file, along with `actual_images_dir` in test_debug2.py
+output = "/lcrc/group/e3sm/ac.forsyth2/zppy_test_debug2_output/pr-400v42/v2.LR.historical_0201"
```

Comment:
I used the `debug2` cfg to test on the v2 simulation used in the original complete_run tests. It's generating the 8 original plots with `v2.LR.historical_0201` data. The visuals can be seen [here](https://web.lcrc.anl.gov/public/e3sm/diagnostic_output/ac.forsyth2/zppy_test_debug2_www/pr-400v42/v2.LR.historical_0201/global_time_series/global_time_series_1850-1860_results/v2_historical_0201_glb_original.pdf).

Error 1) Notice that the plots differ from the [expected results]( https://web.lcrc.anl.gov/public/e3sm/zppy_test_resources/expected_complete_run/global_time_series/global_time_series_1850-1860_results/v2_historical_0201_glb.pdf).  For example, the values printed to the right of the plots differ:
| Plot | debug2.cfg | Expected Results  | 
| --- | --- | --- |
| NET TOA flux (restom)  | 0.81 | 0.16 | 
| Global surface air temperature  | 13.50 | 13.63 |
| TOA radiation | N/A | N/A | 
| Net atm energy imbalance | 0.11 | 0.01 | 
| Change in ocean heat content | +0.15 | +0.15 |
| Max MOC Atlantic streamfunction at 26.5N | N/A | N/A | 
| Change in sea level | -0.2869 | -0.2869 |
| Net atm water imbalance | -0.2036 | -0.0450 |

Interestingly, the plots that get excluded if you set `atmosphere_only = True` have matching values. That implies only the atmosphere plots are different. And this happens even though the relevant cfg parameters are identical.

It's using the just-released Unified (1.9.3) with just-rebased `zppy` code, so I really have no idea why these results would be different. The only thing I can think of is I haven't yet included something important regarding NCO/`ncclimo`. Nevertheless, the diff doesn't show anything in `ts`, so it must be an issue on the `global_time_series` side.

Reply:
The differences with expected values are obviously a showstopper for a release. The latest ncclimo snapshot, 5.2.3-alpha02, could resolve all these differences. We're trying to figure out a way to test that snapshot with `zppy` now. Unfortunately I'm traveling until late tomorrow, when progress on that will resume.

Reply:
Thank you @czender!

Reply:
The expected results were generated during testing for zppy 2.4.0/Unified 1.9.3 -- that is, using NCO as of Unified 1.9.3, the same as used here. So, I'm not entirely sure the new NCO version would fix these issues. But I also can't think of a better explanation.